### ISSUE TITLE
option to add one space padding left and right in column

### ISSUE THREAD

**Issue by tessus** [roman-r-m/joplin-plugin-table-formatter#4]
Is it possible to add an option so that the re-formatting adds a one space padding for each column (left, right)?

Instead of:

```
|head1|head2|head3|
|-----|-----|-----|
|col1 |col2 |col3 |
|col1 |col2 |col3 | 
```

like this:

```
| head1 | head2 | head3 |
|-------|-------|-------|
| col1  | col2  | col3  |
| col1  | col2  | col3  |
```

IMO this way the table is more legible.

**Comment by roman-r-m** [comment]
I agree, it does look better. I think I'll just make it always add the padding.

**Comment by tessus** [comment]
I do also understand that If one has a lot of columns a "compact" setting _could_ make sense, but since a global setting won't really help in that regard, you might as well make it always add the padding.

**Comment by roman-r-m** [comment]
Just published 1.1.2 to npm.

**Comment by tessus** [comment]
Awesome, thanks. FYI: The cron job to build the joplin repo runs every 30 minutes starting at the full hour. (I think I ran into rate limiting issues when doing 15 minutes.)
